starter/package-lock.json

```diff
@@ -1556,40 +1823,41 @@
         "karma-sourcemap-loader": "^0.3.7",
         "karma-typescript": "^4.0.0",
         "karma-typescript-angular2-transform": "^4.0.0",
+        "karma-typescript-es6-transform": "^4.0.0",
         "protractor": "^5.4.2",
         "puppeteer": "^1.11.0"
       },
       "dependencies": {
         "istanbul-lib-coverage": {
-          "version": "2.0.1",
-          "resolved": "https://registry.npmjs.org/istanbul-lib-coverage/-/istanbul-lib-coverage-2.0.1.tgz",
-          "integrity": "sha512-nPvSZsVlbG9aLhZYaC3Oi1gT/tpyo3Yt5fNyf6NmcKIayz4VV/txxJFFKAK/gU4dcNn8ehsanBbVHVl0+amOLA==",
+          "version": "2.0.3",
+          "resolved": "https://registry.npmjs.org/istanbul-lib-coverage/-/istanbul-lib-coverage-2.0.3.tgz",
+          "integrity": "sha512-dKWuzRGCs4G+67VfW9pBFFz2Jpi4vSp/k7zBcJ888ofV5Mi1g5CUML5GvMvV6u9Cjybftu+E8Cgp+k0dI1E5lw==",
           "dev": true
         },
         "istanbul-lib-instrument": {
-          "version": "3.0.0",
-          "resolved": "https://registry.npmjs.org/istanbul-lib-instrument/-/istanbul-lib-instrument-3.0.0.tgz",
-          "integrity": "sha512-eQY9vN9elYjdgN9Iv6NS/00bptm02EBBk70lRMaVjeA6QYocQgenVrSgC28TJurdnZa80AGO3ASdFN+w/njGiQ==",
+          "version": "3.1.0",
+          "resolved": "https://registry.npmjs.org/istanbul-lib-instrument/-/istanbul-lib-instrument-3.1.0.tgz",
+          "integrity": "sha512-ooVllVGT38HIk8MxDj/OIHXSYvH+1tq/Vb38s8ixt9GoJadXska4WkGY+0wkmtYCZNYtaARniH/DixUGGLZ0uA==",
           "dev": true,
           "requires": {
             "@babel/generator": "^7.0.0",
             "@babel/parser": "^7.0.0",
             "@babel/template": "^7.0.0",
             "@babel/traverse": "^7.0.0",
             "@babel/types": "^7.0.0",
-            "istanbul-lib-coverage": "^2.0.1",
+            "istanbul-lib-coverage": "^2.0.3",
             "semver": "^5.5.0"
           }
         }
       }
     },
     "@nationalbankbelgium/stark-ui": {
-      "version": "10.0.0-beta.3",
-      "resolved": "https://registry.npmjs.org/@nationalbankbelgium/stark-ui/-/stark-ui-10.0.0-beta.3.tgz",
-      "integrity": "sha512-3WxUplGqDekaknhK7rwWIPRE092HuR/zs6BD6G3o0/TSx2o820m92u9zWmq2NLyabzfopUapH2lEredVYtgxBw==",
+      "version": "file:../dist/packages-dist/stark-ui/nationalbankbelgium-stark-ui-10.0.0-beta.4-f331991e.tgz",
```

Comment:
Can you revert these 2 lines? They should point to the latest version of Stark on NPM ;)